src/Generating/FinishGen.cpp

```diff
@@ -1769,6 +1769,25 @@ void cFinishGenOreNests::GenerateOre(
 		}
 	}
 
+	// Gold ores are generated more often in Mesa-Type-Biomes:
+	// https://minecraft.gamepedia.com/Gold_Ore
+	if (a_OreType == E_BLOCK_GOLD_ORE)
+	{
+		auto BiomeSampleOne =    a_ChunkDesc.GetBiome( 4,  4);
+		auto BiomeSampleTwo =    a_ChunkDesc.GetBiome( 4, 12);
+		auto BiomeSampleThree =  a_ChunkDesc.GetBiome(12,  4);
+		auto BiomeSampleFour =   a_ChunkDesc.GetBiome(12, 12);
+
+		if ((IsBiomeMesa(BiomeSampleOne)) ||
+			(IsBiomeMesa(BiomeSampleTwo)) ||
+			(IsBiomeMesa(BiomeSampleThree)) ||
+			(IsBiomeMesa(BiomeSampleFour)))
```

Comment:
I guess you could format it like

```
if (
    A ||
    B ||
)
```

with the trailing ) on another line. (actually same for line 1766)

Also it looks like there's extra parentheses around the individual conditions.

Reply:
There is some extra Paranthesis

I thought the styleguide in contributing.md said that

Reply:
The parentheses should be around comparisons, not around simple function calls. So it's 
```cpp
if ((A < B) || (B < C))
```
and
```cpp
if (FunctionCall(A) || FunctionCall(B))
```

Reply:
i also messed up this in the emerald commit so new pull reqeust